_includes/tutorials/kafka-producer-application/kafka/code/src/main/java/io/confluent/developer/KafkaProducerApplication.java

```diff
@@ -81,9 +81,7 @@ public static void main(String[] args) throws Exception {
         final String topic = props.getProperty("output.topic.name");
         final Producer<String, String> producer = new KafkaProducer<>(props);
         final KafkaProducerApplication producerApp = new KafkaProducerApplication(producer, topic);
-
-        // Attach shutdown handler to catch Control-C.
-        Runtime.getRuntime().addShutdownHook(new Thread(producerApp::shutdown));
+        
```

Comment:
The application used to run indefinitely until the user this CTRL+C, but now it will produce the records in a file and shutdown, so I've removed the shutdown hook.